test/server/filter_chain_benchmark_test.cc

```diff
@@ -102,6 +102,18 @@ class MockConnectionSocket : public Network::ConnectionSocket {
   void addOptions(const OptionsSharedPtr&) override {}
   const OptionsSharedPtr& options() const override { return options_; }
   void setRequestedServerName(absl::string_view) override {}
+  Api::SysCallIntResult bind(Network::Address::InstanceConstSharedPtr address) override {
+    (void)address;
```

Comment:
nit: just delete the param name from the variable, or used UNREFERENCED_PARAMETER, same elsewhere.

Reply:
Ow, thanks for that!